src/data/index.ts

```diff
@@ -475,6 +476,13 @@ const json: RequirementsJson = {
       advisors: appliedMathMinorAdvisors,
       abbrev: 'AppliedMath',
     },
+    ATMOSCI: {
+      name: 'Atmospheric Science',
+      schools: ['EN'],
```

Comment:
```suggestion
      schools: ['AG', 'EN'],
```

Need to verify this further, but as the Department of Earth and Atmospheric Sciences seems to be in both Cornell Engineering and CALS, we probably need to add CALS to this list.

https://www.eas.cornell.edu/eas/programs/undergraduate-programs/undergraduate-minors/atmospheric-science-minor